test-it/ComplexMathFunctions.via

```diff
@@ -1,21 +1,150 @@
 
-
 define(HelloComplex dv(.VirtualInstrument (
     Locals: c(
         //-------------------
         e(dv(.Double 2.0) dMagnitude)
         e(dv(.Double 0.5) dPhase)
+        e(dv(.Double 0.0) dReal)
```

Comment:
Should we add tests where input to the supported complex operations is special floating point numbers such as Infinity, -Infinity, NAN, etc?